docs/development/setup-advanced.md

```diff
@@ -58,6 +58,16 @@ source /srv/zulip-py3-venv/bin/activate
 ./tools/run-dev  # starts the development server
 ```
 
+**Note**: If you're installing into a Linux Container, be sure to (1) set the `EXTERNAL_HOST` environment variable before starting the development server, and (2) use the `--interface` argument of the run-dev command. For example, if the IPV4 address of your Linux Container is 10.246.213.217, the above commands would instead look like this:
```

Comment:
This should be line-wrapped; I'm surprised `prettier` didn't do it for you. 

I think we also have a tip syntax you can use for this block? 

Also we can link to docs/development/remote.md; it might be the code block you have below belongs on that page.

(And maybe we should edit the description of `remote`, above, to say that a local container is effectively the same as a remote environment for the purpose of these instructions)

Reply:
Code updated.